README.md

```diff
@@ -123,8 +158,68 @@ It includes:
 More details about vetters can be found in the individual vetters package
 documentation.
 
+<br>
+<br>
+
 ## Contributing
 Individuals or business entities who contribute to this project must have
 completed and submitted the [F5® Contributor License Agreement](https://github.com/aspenmesh/cla/raw/master/f5-cla.pdf)
 to [cla@aspenmesh.io](mailto:cla@aspenmesh.io) prior to their code submission
 being included in this project. Please include your github username in the CLA email.
+
+### Build Prerequisites
+To build Istio-Vet locally, you will need to install the following:
+
+* A [Go environment](https://golang.org/doc/install).
+* Install [dep](https://golang.github.io/dep/docs/installation.html)
+* Install [Protobuf](https://github.com/golang/protobuf)
+  * Protobuf has a few installation steps. First, download the [protocol-buffers](https://developers.google.com/protocol-buffers/) and note:
+    ```For non-C++ users, the simplest way to install the protocol compiler is to download a pre-built binary from our release page:
+    https://github.com/protocolbuffers/protobuf/releases
+    ```
+    Mac users can use Brew.  Linux users can get the release with this command. Make sure to change the protoc version and filename:
+    ```bash
+    curl -L -O \
+    https://github.com/google/protobuf/releases/download/<desired-version>/protoc-<desired-version>-linux-x86_64.zip \
+    && mkdir -p /usr/local \
+    && unzip protoc-<desired-version>-linux-x86_64.zip -d /usr/local
+    ```
+    You should be able to type `protoc` at the command line now and see its options.
+    Next, get protobuf:
+    ```bash
+    go get -u github.com/golang/protobuf/protoc-gen-go
+    ```
+* Install [Mercurial](https://www.mercurial-scm.org/) if not already installed. Mac users can use Brew. Linux users can run `apt-get install mercurial`
```

Comment:
Verify if we need mercurial.

Reply:
Our official Istio-Vet image doesn't have it, so I'm taking it out. :)